test/fixtures/plugin.legend/navigation/legend-navigation-horizontal-multiple-lines.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Yes, that's expected, because it only has one line, so personally I think it looks like the navigation is being done horizontally, not vertically like when we have multiple lines. But I can change that if you think it's better.